### PR TITLE
Shuffle install test

### PR DESCRIPTION
**Pull request by unnonouno**
I replaced installation test with shuffled test. It takes only one ID and makes configuration based on the ID.
merge #247 first

### CODE REVIEW

File: run_cupy_install_test.py [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
 import docker
+import shuffle
 
 
-def append_requires(requires, name, ver):
-    if ver and ver != 'none':
-        requires.append('%s==%s' % (name, ver))
+params = {
+    'base': docker.base_choices,
+    'cuda': docker.cuda_choices,
```
Comment (okuta) [inline run_cupy_install_test.py]: ` docker.cuda_choices` includes `'none'`.
I got error when I run `./run_cupy_install_test.py --id 0`.

**Comment by okuta** [comment]
I got error when I run `./run_cupy_install_test.py --id 0`.
```
Step 10 : RUN touch /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn.so
 ---> Running in dbbf49fe49d4
touch: cannot touch '/usr/local/cuda/include/cudnn.h': No such file or directory
touch: cannot touch '/usr/local/cuda/lib64/libcudnn.so': No such file or directory
The command '/bin/sh -c touch /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn.so' returned a non-zero code: 1
```

**Comment by unnonouno** [comment]
I fixed it

File: run_cupy_install_test.py (hunk 1 above)
Comment (unnonouno) [inline run_cupy_install_test.py]: Ah, i need to remove it.

**Comment by okuta** [comment]
LGTM!

Can I merge this PR?

**Pull request merged**